### ISSUE TITLE
Option to create shard.yml interactively 

### ISSUE THREAD

**Issue by dinko-pehar** [crystal-lang/shards#542]
NPM and some other package managers have the ability to generate their versions of `shard.yml` file based on user provided input.
Similar to those package managers, when we execute `shards init --interactive`, it should ask user for basic input, such as package name, author name etc. The fill in provided data into the `shard.yml`

Where `shard init` can generate basic `shard.yml` file, maybe this option can ask for more details of everything that can go inside `shard.yml`, such as _target_, _homepage_, _repository_ etc

**Comment by m-o-e** [comment]
Possible alternative/related:

* Write all available options into the generated `shard.yml`
  (atm at least `crystal: <version>` is missing, possibly others)
  
* Unify `shards init` and `crystal init`?
   Might be just me but I often type the former by accident when I actually meant the latter.

Personally I don't see a need for interactive prompting since `shard.yml`
is so easy to write (unlike e.g. `package.json` in npm).

**Comment by dinko-pehar** [comment]
@m-o-e I agree it's a simple file to write (`shard.yml`). Maybe in future, if this file contained more information (_all package manager files at one point contain lot of info_), it would be nice to have it created interactively. On the other hand, I see some of the described keywords in `man shards` are missing on `shard init`. This is not important feature at the moment.
